### ISSUE TITLE
NVDA Does Not Allow for Text Reading and Navigation in Windows Live Writer's Edit Boxes

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4160]
**Reported by dgoldfield on 2014-05-29 16:20**
When using NVDA with the latest version of Windows Live Writer, I am unable to use the cursor keys to identify the location of the caret in edit fields while trying to compose a blog entry.  I have little difficulty in the actual setup screen.

**Comment by ehollig** [comment]
Sounds like Windows Live Writer end-of-life was in 2012 acording to [Wikipedia](https://en.wikipedia.org/wiki/Windows_Live_Writer). An open source fork was released as Open Live Writer but to my knowledge, no issues have been raised about this. Can this be closed?

**Comment by leonardder** [comment]
Duplicate of #1709
